src/app/3d/qgspointcloud3dsymbolwidget.cpp

```diff
@@ -569,3 +570,22 @@ void QgsPointCloud3DSymbolWidget::maximumScreenErrorChanged( double maxScreenErr
   mMaximumScreenError = maxScreenError;
   emitChangedSignal();
 }
+
+void QgsPointCloud3DSymbolWidget::showBoundingBoxesChanged( int checkBoxState )
+{
+  Q_UNUSED( checkBoxState );
+  bool showBoundingBoxes = mShowBoundingBoxesCheckBox->isChecked();
+  if ( showBoundingBoxes == mShowBoundingBoxes )
+    return;
+  mShowBoundingBoxes = showBoundingBoxes;
+  emitChangedSignal();
```

Comment:
If we get rid of `mShowBoundingBoxes`, we could reduce body of this function to `emit changed();` - or even better, remove the function and connect QCheckBox::stateChanged to "changed" signal.